.github/workflows/quality-check-dagster-notdiamond.yml

```diff
@@ -0,0 +1,39 @@
+name: quality-check
```

Comment:
For the quality check, and build and release workflows, you should be able to copy:

- `quality-check-dagster-contrib-modal.yml`
- `release-dagster-contrib-modal.yml`

It looks like you copied the template itself, whereas the above invoke it like so:

```
jobs:
  build-and-release-dagster-contrib-modal:
    uses: ./.github/workflows/template-release.yml
    with:
      library_name: dagster-contrib-modal
      working_directory: ./libraries/dagster-contrib-modal
    secrets: inherit
```